pages/windows/vcvarsall.md

```diff
@@ -0,0 +1,21 @@
+# vcvarsall
```

Comment:
Wait a moment. This has to *exactly* match the name of the command. Same goes for the filename.

Reply:
```suggestion
# vcvarsall.bat
```

Reply:
Do you specify exe for all the Windows commands? If not, I should remove bat here too

Reply:
But in Windows `.bat` is optional, I think we shouldn't use `.bat` in the examples.

Reply:
Fixed in https://github.com/tldr-pages/tldr/pull/7439/commits/cd8a83afd56368e307ab948627f6f47f328d1970